readme.md

```diff
@@ -25,14 +25,26 @@ Example file names and locations for an image widening transform:
 * **test/WidenImage.js** - tests of the transform. *required*
 * **test/fixtures/WidenImage.html** - fixtures used by transform tests. *optional*
 
-### Changing styles
-- Prefer to modify classlist rather than style attributes. For example, **WidenImage.js** mentioned above could add/remove a class name from **WidenImage.css** to an element's classlist to help achieve image widening.
-
-### Function naming
-*Coming soon.*
-
-### Style naming
-*Coming soon.*
+Directory names should be lowercase.
+
+### Functional
+- Prefer to read and modify the CSS class list rather than style attributes or
+  computed styles. This allows clients to customize appearance with CSS
+  overrides and no JavaScript changes or effects. For example, **WidenImage.js**
+  mentioned above could add/remove a class name from **WidenImage.css** to an
+  element's class list to help achieve image widening.
+- Prefer Minerva and Parsoid style defaults. Any deviations should be deliberate
+  and tightly scoped
+- Avoid usage of `!important` in stylesheets
```

Comment:
Move to first bullet point and reword to: Prefer to avoid usage of `!important` in stylesheets. However, if given the ultimatum of using a class with `!important` or an inline style, prefer the class with `!important`.